README.md

```diff
@@ -5,77 +5,87 @@
 
-We are planning to require teams to keep a daily log of short updates about
-their work. Our idea is to allow any sort of blog type tool for that (maybe
-recommend a few) and aggregate things through RSS in a central app. This app
-could then act as a webhook target for GitHub events.
-
-Since we still are somewhat overwhelmed with the amount of work we'd like to
-ask the community for help with this. The app would need to be available
-(initial, basic version) on 1st of July, ideally a few days earlier.
-
 Features:
 
-* Users can sign in through GitHub Oauth
-* They can create and update teams
-* Teams have members (students, coaches, mentors aka project maintainers), GitHub repositories, a log URL
-
+* Users can sign in through GitHub OAuth
+* Students can create teams and apply for the program
+* Teams can organize their members (students, coaches, mentors aka project maintainers)
 * RSS feeds are fetched from all teams' logs regularly and aggregated
-* There is a webhook endpoint for GitHub events that aggregates information about issues, pull requests and such
+* Organizers can process new projects for submission, review new teams for selection, keep track of information on conferences for students
+
+We're really excited at how much this app has grown and developed, and are really
+grateful for all of the community help along the way. There's still plenty to do
+here - and we'd love to have your contributions! Hop on over to the
+[issues](https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues)
+and have a look 👀.
 
 Requirements:
 
-* Keep it simple, so Rails Girls students can get involved, too
+* Keep it simple to lower the barrier for RGSoC students to contribute too
 * By contributing, you agree to adhere to our [Code of Conduct](CODE_OF_CONDUCT.md)
 * Make sure to check our [Contribution Guide](CONTRIBUTING.md)
 
-## System Requirements
+## Getting Started
+
+In order to contribute, you need to run the Teams App locally on your computer. For setting it up, you'll need to have some tools installed on your system:
 
 * PostgreSQL 9.5 or newer
 * Ruby 2.5.1
 * [Chromedriver](https://sites.google.com/a/chromium.org/chromedriver/)
 
-### Setup on Ubuntu
+The following section will guide you step by step through the setup and installation process on Linux 🐧 and macOS 🍏
+
+### 1. Installation
+
+#### Setup on Ubuntu 🐧
 ```bash
 # Install required packages
-$ sudo apt-get install postgresql libpq-dev libcurl3 libcurl3-gnutls libcurl4-openssl-dev postgresql-contrib-9.5 chromium-chromedriver
+sudo apt-get install postgresql libpq-dev libcurl3 libcurl3-gnutls libcurl4-openssl-dev postgresql-contrib-9.5 chromium-chromedriver
+
 # Create database user rgsoc with password rgsoc
-$ sudo -u postgres createuser -P -s rgsoc
-Enter password for new role: rgsoc
-Enter it again: rgsoc
+sudo -u postgres createuser -P -s rgsoc
+# Enter password for new role: rgsoc
+# Enter it again: rgsoc
```

Comment:
Maybe we could add the same `#`s on the 🍏 instructions for consistency 👍

Reply:
Well, for the sake of _"consistency"_ probably yes, you're right - but for the sake of _"simplicity"_ no I'd say 😉 

Installing `postgres` via homebrew automatically generates a password-less role with the same name as the user logged in into the system -> that's exactly what Rails defaults to for a database `username` and `password` if it's skipped in the `database.yml` file.

The `database.yml.example` file has exactly this configuration - so it would work outta the box without modifying it.

In my opinion ☝️ is just easier, therefore I did not add any `role` instructions.

For 🐧 tbh, they're also just there because someone who works with Linux once specified them like this. Maybe it's also obsolete there... 🤔

Reply:
Good point!

I was however just wondering: can we not just ditch the whole `role` thing? If one installs `postgres` via homebrew, it will automatically create a password-less role with the same name as the user on the system -> that's exactly what Rails defaults to as `username` and `password` if it's not specified explicitly in the `database.yml` file (as the default `database.ym file and our `database.yml.example` also do).

Not sure if the same works for 🐧 though 🤔

Reply:
I think so! I tried it on my VM and it worked, but I needed to remove `host: localhost` for it to work. Otherwise it was requiring a password, even if was trying to create a database from scratch using `rails db:setup`.

This may give some context into what's going on: https://stackoverflow.com/questions/23375740/pgconnectionbad-fe-sendauth-no-password-supplied

It may be the case that by default on linux `localhost` is set to only allow md5 connections, while on macOS it doesn't. It seems to be the case, looking at my own configurations.

```
# On Ubuntu:
host    all             all             127.0.0.1/32            md5

# On macOS:
host    all             all             127.0.0.1/32            trust
```

Since our `database.yml.example` has `host: localhost`, people on linux may see some errors 😟

Here's what I was seeing when trying to create the DB:

```
pedro@ubuntu:~/code/rgsoc-teams$ bundle exec rails db:setup
fe_sendauth: no password supplied
Couldn't create database for {"adapter"=>"postgresql", "database"=>"rgsocteams_development_2", "host"=>"localhost"}
rails aborted!
PG::ConnectionBad: fe_sendauth: no password supplied
bin/rails:4:in `<main>'
Tasks: TOP => db:setup => db:schema:load_if_ruby => db:create
(See full trace by running task with --trace)
```

My settings were:

```
development:
  adapter: postgresql
  database: rgsocteams_development_2
  host: localhost
```

The same settings work on macOS.

Reply:
Oh, you're so right I totally forgot about this 🙈 

Removing the `host` key sounds more of a hack / coincidence than an actual setting, so I think we're closed to a default that people are used to if we just stick to creating a role with a password.

I'll update the section accordingly ✌️